### PR TITLE
Update manage-user-profiles.md

### PR DESCRIPTION
**Pull request by DamianWiese**
Added a note to the section "Add and remove admins for a user's OneDrive" indicating the lack of support for removing site collection administrator rights for their OneDrive for Business site. Please see the internal Yammer thread below for a discussion on this topic:
https://www.yammer.com/microsoft.com/#/threads/show?threadId=686785069760512

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-user-profiles.md [hunk 1]
```diff
@@ -70,7 +70,9 @@ Follow these steps to transfer ownership of a OneDrive to a different user, or g
     ![Manage the owners of a OneDrive](media/120f7c8c-262f-4a41-a484-e830c662f534.png)
   
 For info about automatically transferring ownership of OneDrive to a user's manager when the user account is marked for deletion, see [Set up access delegation](/onedrive/retention-and-deletion).
-  
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-user-profiles.md:73]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/manage-user-profiles.md [hunk 2]
```diff
@@ -70,7 +70,9 @@ Follow these steps to transfer ownership of a OneDrive to a different user, or g
...
+>For full OneDrive for Business functionality the user must be a site collection administrator of their OneDrive for Business site. It is not supported to remove this permission.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-user-profiles.md:74]:
```suggestion
> For full OneDrive functionality, the user must be a site collection administrator of the organization's OneDrive site. Removing this permission is not supported.
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/manage-user-profiles.md:74]: Please do not use OneDrive for Business.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kaarins** [comment]
@DamianWiese thank you for this! I already added a warning when I was making other changes to user profile content.

**Pull request closed without merging**